packages/api/cms-api/src/auth/guards/comet.guard.ts

```diff
@@ -38,7 +38,11 @@ export class CometAuthGuard implements CanActivate {
         }
 
         const result = await this.getAuthenticatedUserResult(request);
-        if (!result) return false;
+        if (!result) {
+            const location = `${context.getClass().name}::${context.getHandler().name}()`;
+            console.log(`CometAuthGuard: No AuthService could authenticate the user for ${location}.`);
```

Comment:
Instead of console.log I'd use a Logger, like it's done here: https://github.com/vivid-planet/comet/blob/main/packages/api/cms-api/src/kubernetes/prevent-local-invocation.guard.ts#L13. Then we don't need to specify the guard's name in the log call.